db/migrate/20230804193208_add_confirmable_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddConfirmableToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmation_token, :string
+    add_index :users, :confirmation_token, unique: true
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string
```

Comment:
- [ Required ] I think it would be better if you delete this migration file from your project. I noticed that this migration file tries to add the `confirmation_token` column to the `users` table which causes the migration to cause an error because the `confirmation_token` column already exists on the `users` table courtesy of the `20230804150918_add_devise_to_users.rb` migration file. Please see the screenshot below for more details on this and rectify it. 

## Error running add_confirmable_to_users migration file
![Screenshot from 2023-08-04 23-00-49](https://github.com/massazoum/Rails_App/assets/16392046/87d5391a-82ee-4ee8-85cf-9e2be617606c)